client/tests/conftest.py

```diff
@@ -52,9 +53,20 @@ def configured_logger() -> BoundLogger:
     return structlog.get_logger("nublado-client")
 
 
+def _create_mock_token(username: str, token: str) -> str:
+    # A mock token is: "gt-<base-64 encoded username>.<base64 encoded token>"
+    # That is then decoded to extract the username in the Jupyter mock.
+    enc_u = urlsafe_b64encode(username.encode()).decode("utf-8")
+    enc_t = urlsafe_b64encode(token.encode()).decode("utf-8")
```

Comment:
```suggestion
    enc_u = urlsafe_b64encode(username.encode()).decode()
    enc_t = urlsafe_b64encode(token.encode()).decode()
```
UTF-8 is the default.